sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java

```diff
@@ -38,6 +40,18 @@ public AnalyzeHealthcareEntitiesOptions setIncludeStatistics(boolean includeStat
         return this;
     }
 
+    /**
+     * Get the value of {@code disableServiceLogs}.
```

Comment:
`disableServiceLogs` is private and the user will not see this field. Instead, add javadoc to specify what that field represents. "Returns "true" if the service logs are disabled"